auth/api-client/snippets.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to authenticate to Google Cloud Platform APIs using
+the Google API Client."""
+
+import argparse
+
+
+def implicit(project):
+    import googleapiclient.discovery
+
+    # If you don't specify credentials when constructing the client, the
+    # client library will look for credentials in the environment.
+    storage_client = googleapiclient.discovery.build('storage', 'v1')
+
+    # Make an authenticated API request
+    buckets = storage_client.buckets().list(project=project).execute()
+    print(buckets)
```

Comment:
Okay, my preference is to include the import because print with parens is confusing syntax in Py2, but I'll leave it to you.